<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebSearchView</class>
 <widget class="QMainWindow" name="WebSearchView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1227</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1227</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WebSearchView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1227</width>
     <height>663</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1227</width>
     <height>663</height>
    </size>
   </property>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="URLEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>631</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="URLLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ThreadsSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ThreadsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of threads</string>
    </property>
   </widget>
   <widget class="QLabel" name="SearchLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Text to search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>631</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="URLNumLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max number of URLs</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="URLSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tasksStatuses">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>1191</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
